src/components/list-item/ListItem.stories.tsx

```diff
@@ -0,0 +1,156 @@
+import { ListItem } from "./ListItem";
+import { IconChevronRight, IconDragable, IconGearSettings, IconPin, IconStarOutlined } from "../..";
+import { useState } from "react";
+
+function ListItemDemo({ children, activated, onClick = undefined, ...props }) {
+  return (
+    <div {...props} style={{ width: "fit-content", minWidth: 320 }}>
+      <ListItem activated={activated} onClick={onClick}>
+        {children}
+      </ListItem>
+    </div>
+  );
+}
+
+export default {
+  title: "Components/Interactional/ListItem",
+  component: ListItemDemo,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/1hz5D4Q7pj5w0YrRw7hXbu/UI-Kit?node-id=19%3A861",
+    },
+  },
+};
+
+const SmallListItemTemplate = ({ showLeftIcon, showRightIcon, activated, content }) => (
+  <ListItemDemo activated={activated}>
```

Comment:
[] If your component has a value it also needs a way to set that value, pass an onChange event.